compiler/internal/codegen/codegen_main.go

```diff
@@ -142,6 +144,21 @@ func (b *Builder) computeHandlerRegistrationConfig(mwNames map[*est.Middleware]*
 	})
 }
 
+func (b *Builder) computeServiceInitConfig() *Statement {
+	return Index().Qual("encore.dev/appruntime/service", "Initializer").CustomFunc(Options{
+		Open:      "{",
+		Close:     "}",
+		Separator: ",",
+		Multi:     true,
+	}, func(g *Group) {
```

Comment:
I'd prefer we use `Values` rather `custom` for these types of blocks, and then construct the `[]Code` for the values

Reply:
This is ensuring we have newlines for each entry rather than putting all on the same line. Or is there a better way to accomplish that?